src/platform/cordova.scss

```diff
@@ -44,9 +44,6 @@ $cordova-statusbar-padding-modal-max-width:         767px !default;
 
   > .toolbar.statusbar-padding:first-child {
     padding-top: calc(#{$cordova-statusbar-padding} + #{$toolbar-padding});
-
-    height: calc(#{$toolbar-height} + #{$cordova-statusbar-padding});
-    min-height: calc(#{$toolbar-height} + #{$cordova-statusbar-padding});
```

Comment:
no, it is not. Try yourself.

When the statusbar is visible, we need to change the min-height as well, since the total height is `content height + padding` since the box-sizing mode is border-box. This probably could be improved by using `content-box`, but in the meantime, changing the `min-height` is needed.